doc/tutorials/nvmx_virtualization.rst

```diff
@@ -71,6 +71,20 @@ hypervisor to actually run the L2 guest.
    a. L0 reflects most VMEXits to L1 hypervisor
    #. L0 runs L1 guest with VMCS01 and VMCS02 as the shadow VMCS
 
+VPID allocation
+===============
+
+ACRN doesn't emulate L2 VPIDs and allocates VPIDs for L1 VMs from the reserved top
+16-bit VPID range (`0x10000U - CONFIG_MAX_VM_NUM * MAX_VCPUS_PER_VM` and up).
```

Comment:
```suggestion
16-bit VPID range (``0x10000U - CONFIG_MAX_VM_NUM * MAX_VCPUS_PER_VM`` and up).
```

Reply:
@dbkinder thank you very much for your comments! I will get them fixed.